bin/db-migrate

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 printf "Migrating database ..\n" \
```

Comment:
"Migrating database" and "Upgrading database" messages seems to be confusing - they sound like the same thing.
What about "Creating migration" and "Upgrading database" ?

Reply:
renamed